apps/dashboard/app/views/batch_connect/session_contexts/_path_selector.html.erb

```diff
@@ -39,20 +39,25 @@
           <span aria-hidden="true">&times;</span>
         </button>
       </div>
-      <div class="modal-body">        
+      <div class="modal-body">       
         <div>
-            <ol id="<%= breadcrumb_id %>" class="breadcrumb breadcrumb-no-delimiter">
-            </ol>
-            <table class="table table-hover table-condensed d-table w-100" id="<%= table_id %>">
-                <thead>
-                  <tr>
-                      <th>Type</th>
-                      <th>Name</th>
-                  </tr>
-                </thead>
-                <tbody>
-                </tbody>
-            </table>
+          <ol id="<%= breadcrumb_id %>" class="breadcrumb breadcrumb-no-delimiter">
+          </ol>
+          <div class="d-flex justify-content-center">
+            <div id="loading-icon" class="spinner-border spinner-grow spinner-grow-sm" role="status" style="width: 5rem;height: 5rem;">
```

Comment:
`spinner-grow spinner-grow-sm` are cool - but I think we want to coalesce around just regular spinners (they're used in other places, so I just want it all to look the same). 

Also - try to avoid in-lining `style`s like this. Eventually, we'll have a CSP that'll disable this. Maybe we just need a `.rem-5` class?

https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy/style-src#unsafe_inline_styles